mysql/datadog_checks/mysql/mysql.py

```diff
@@ -179,6 +191,7 @@ def check(self, _):
                     tags = tags + self._get_runtime_aurora_tags(db)
 
                 self.check_performance_schema_enabled(db)
+                self.check_userstat_enabled(db)
```

Comment:
instead of calling this method on every check run, can we only call it if the user enables the `table_rows_stats_metrics` config option?

Reply:
yeah, but this setting only applies to a subset of mysql engines where `performance_schema` applies across the board.

Theres no need to check if this setting is enabled, unless the user wishes to collect these metrics and if they do the method will be executed on every check run anyways, right?

Reply:
Ok ! I added a check in https://github.com/DataDog/integrations-core/pull/11043/commits/2ec9916d9b74b14a7455005695f7aeefbbd88712 to handle this :)

Reply:
thanks! 😄